src/Assignments/ProblemIframe.tsx

```diff
@@ -67,8 +69,35 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
         // when problem changes, reset lastsubmitted and lastsaved
         setLastSubmittedAt?.(null);
         setLastSavedAt?.(null);
+        setLastSubmission({});
     }, [problem.id]);
 
+    const updateSubmitActive = _.throttle(() => {
+        const submitButtons = iframeRef.current?.contentWindow?.document.getElementsByName('submitAnswers') as NodeListOf<HTMLButtonElement>;
+        const problemForm = iframeRef.current?.contentWindow?.document.getElementById('problemMainForm') as HTMLFormElement;
+        // called only onLoad or after interaction with already loaded srcdoc - so form will exist, unless bad problemPath
+        // no console.error because exam problems (and static problems) will not have 'submitAnswers'
+        if (_.isNil(submitButtons) || _.isNil(problemForm)) {return;}
```

Comment:
FYI you don't need the brackets

Reply:
I personally always like having the brackets but I also put it on a newline, @PanopticaRising would disagree with me though I think

Reply:
looks like I'll split the difference